src/app/components/challenge/challengesubmissions/challengesubmissions.component.html

```diff
@@ -63,108 +63,94 @@
+          </td>
+        </ng-container>
+        <ng-container matColumnDef="expandedDetail">
+          <td mat-cell *matCellDef="let element" [attr.colspan]="columnsToDisplay.length">
+            <div class="elements" [class.expanded-row]="element == expandedElement"
+                 [@detailExpand]="element == expandedElement ? 'expanded' : 'collapsed'">
+              <div class="elements-detail">
+                <div class="element">
+                  <span class="element-description-attribution">Team Name</span>
+                  <span class="fw-regular element-description"> {{element.participant_team_name}} </span>
+                </div>
+                <div class="element">
+                  <span class="element-description-attribution">Method Name</span>
+                  <span class="fw-regular element-description"> {{element.method_name}} </span>
+                </div>
+                <div class="element">
+                  <span class="element-description-attribution">Method Description</span>
+                  <span class="fw-regular element-description"> {{element.method_description}} </span>
+                </div>
+                <div class="element">
+                  <span class="element-description-attribution">Project Url</span>
+                  <span class="fw-regular element-description"> {{element.project_url}} </span>
+                </div>
+                <div class="element">
+                  <span class="element-description-attribution">Publication Url</span>
+                  <span class="fw-regular element-description"> {{element.publication_url}} </span>
+                </div>
+              </div>
+
+              <div class="elements-action">
+                <div class="element">
+                  <span class="element-description-attribution">Status</span>
+                  <span class="fw-regular element-description" [ngClass]="element.status"> {{element.status}} </span>
+                </div>
+                <div class="element">
+                  <span class="element-description-attribution">Execution Time</span>
+                  <span class="fw-regular element-description"> {{element.execution_time}} </span>
+                </div>
+                <div class="element">
+                  <span class="element-description-attribution">Submitted At</span>
+                  <span class="fw-regular element-description"> {{element.submitted_at | date:'medium'}} </span>
+                </div>
+                <div class="element">
+                  <mat-checkbox [checked]="element.is_public" [disabled]="element.status !== 'finished' || !selectedPhase['leaderboard_public']" (change)="changeSubmissionVisibility(element.id, element.is_public)">
+                    Show On Leaderboard
+                  </mat-checkbox>
+                  <a (click)="editSubmission(element)" class="element-description">
+                    <i class="fa fa-pencil"></i>
```

Comment:
Should we move the edit button to the next line after "Show On Leaderboard"?

Reply:
done